src/Symfony/Installer/NewCommand.php

```diff
@@ -415,13 +415,19 @@ protected function generateComposerProjectName()
      */
     private function fixComposerPackageName($name)
     {
-        return strtolower(
-            preg_replace(
-                array('/([A-Z]+)([A-Z][a-z])/', '/([a-z\d])([A-Z])/'),
-                array('\\1-\\2', '\\1-\\2'),
-                strtr($name, '-', '.')
-            )
+        $name = preg_replace(
+            array('/([A-Z]+)([A-Z][a-z])/u', '/([a-z\d])([A-Z])/u'),
+            array('\\1-\\2', '\\1-\\2'),
+            strtr($name, '-', '.')
         );
+
+        if (function_exists('mb_strtolower')) {
+            $name = mb_strtolower($name);
+        } else {
+            $name = utf8_encode(strtolower(utf8_decode($name)));
```

Comment:
The problem may be the name of the project, but especially the username added by `generateComposerProjectName` (in my case $_SERVER['USERNAME'])

And I tried, this change has not resolved the error.

Otherwise, I don't think using the username for the vendor is really necessary ... a quick solution would be to use only the last condition:

    } else {
        // package names must be in the format foo/bar
        $name = $name.'/'.$name;
    }

... or to replace all special chars (I don't think tha composer package name should contain accented chars)